libprecompiled/Precompiled.h

```diff
@@ -80,8 +80,13 @@ class Precompiled : public std::enable_shared_from_this<Precompiled>
         return m_precompiledExecResultFactory;
     }
 
+    // for UT
+    static void clearName2SelectCache();
+
 protected:
     std::map<std::string, uint32_t> name2Selector;
+    // global function selector cache
+    static tbb::concurrent_unordered_map<std::string, uint32_t> s_name2SelectCache;
```

Comment:
I think this can be deleted, just a static s_name2SelectCache in cpp is fine

Reply:
ok~I will fix it.